cli/ballerina-tool/src/main/resources/cli-help/ballerina-install.help

```diff
@@ -0,0 +1,17 @@
+NAME
+    Ballerina install - Install a given SDK version
+
+SYNOPSIS
+    ballerina install <sdk-version>
+
+DESCRIPTION
+     Install the given Ballerina version from the project repository.
+
+OPTIONS
+     The `ballerina install` command does not accept any options.
+
+EXAMPLES
+     Install Ballerina 1.0.0 version:
+     $ ballerina install 1.0.0
+
+     This will download and keep the Ballerina 1.0.0 distribution and set the Ballerina commands.
```

Comment:
What does it mean "set the Ballerina commands"?

Reply:
Removed the "ballerina-install.help" file as it is not applicable now.